_git/git_commands.py

```diff
@@ -0,0 +1,434 @@
+from aenea import Text
```

Comment:
I split the grammar into two files, so the user will have to make sure to copy both files to the macro system folder on the VM . I think it is necessary to keep the files in a manageable size

Is this okay with you?

Reply:
Fine by me.